Phenotypes.pm

```diff
@@ -91,11 +95,20 @@ use base qw(Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin);
 
 # default config
 my %DEFAULTS = (
-  exclude_sources => 'HGMD_PUBLIC&COSMIC',
+  exclude_sources => 'HGMD-PUBLIC&COSMIC',
   exclude_types => 'StructuralVariation&SupportingStructuralVariation',
   expand_right => 100000,
+  phenotype_feature => 0,
 );
 
+my $out_txt = 1;
```

Comment:
Would it make sense to build a hash instead of creating 3 different variables ?
This would make it easier to maintain if we add a new output format to VEP.
e.g.:
```
my %output_format;
...
# From line 123:
# get output format
if ($self->{config}->{output_format}) {
 $output_format{$self->{config}->{output_format}} = 1;
}
$out_txt = (%output_format) ? 0 : 1;
$char_sep = "+" if ($output_format{'vcf'});
```

Reply:
Thanks for checking @ens-lgil. Good suggestion! Will make an update.